onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -139,6 +154,10 @@ class ConvAddRelu : public NodeSelector {
     if (node_ep != kCudaExecutionProvider) {
       return std::nullopt;
     }
+    if (node.Name() == "Conv_3009")
```

Comment:
Remove this debug code?

Reply:
Oh sorry ! Completely overlooked that. I removed it.